### PR TITLE
update wording of views

### PR DESCRIPTION
**Pull request by Eskibear**
Now we have more views than before, so:
- change view container from "Spring" to "Spring Boot Dashboard"
- change view of apps from "Spring Boot Dashboard" to "Apps".

### CODE REVIEW

**Comment by Eskibear** [comment]
Previous:

image



This PR:

image

**Comment by Eskibear** [comment]
@martinlippert what do you think of the wording change?

**Comment by martinlippert** [comment]
> @martinlippert what do you think of the wording change?

Makes a lot of sense to me! 👍🏻

File: test/suite/extension.test.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ suite("Extension Test Suite", () => {
     });
 
     test("Can view static beans and mappings", async () => {
-        await vscode.commands.executeCommand("spring-boot-dashboard.focus");
+        await vscode.commands.executeCommand("spring.apps.focus");
```
Comment (Eskibear) [inline test/suite/extension.test.ts:20]: @cscherryy I don't understand why renaming would break the case. Can you help to take a look?

**Review by CsCherrYY — APPROVED** [review]

**Pull request merged**